PSK.UII/ClientApp/src/components/Registration/RegistrationPage.js

```diff
@@ -33,8 +33,7 @@ class RegistrationPage extends React.Component {
         } = this.state;
 
         post('registration', {
-            firstName: firstName,
-            lastName: lastName,
+            fullName: firstName + " " + lastName,
             password: password,
             repeatedPassword: repeatedPassword,
```

Comment:
Kam šitas reikalingas? Paprastai slaptažodžio pakartojimas yra skirtas tam, kad žmogus žinotų, kad įrašė tą ką mano, kad įrašė. Mes tą galim patikrint front'e, nereikia to siųst į back'ą